_posts/2018-01-30-omero-5-4-3.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Release of OMERO 5.4.3
+intro-blurb: The OME team announces the release of OMERO 5.4.3.
+---
+Today we are releasing OMERO 5.4.3.
+
+This release is the same as [5.4.2]({{ site.baseurl }}/2018/01/23/omero-5-4-2.html) but with a fix in place for the OMERO.web
+issue.
```

Comment:
Perhaps link "the ... issue" to https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/01/26/omero-5-4-2-issue.html ?